drawee/src/main/java/com/facebook/drawee/drawable/ArrayDrawable.java

```diff
@@ -370,4 +372,14 @@ public void getRootBounds(RectF bounds) {
       bounds.set(getBounds());
     }
   }
+
+  @Override
+  @TargetApi(Build.VERSION_CODES.LOLLIPOP)
+  public void setHotspot(float x, float y) {
+    for (Drawable drawable : mLayers) {
```

Comment:
Please change this foreach to a for loop over the array as with every other method here which loops through the drawables.